modules/olm-mirroring-catalog.adoc

```diff
@@ -94,29 +85,97 @@ src image has index label for database path: /database/index.db
 using database path mapping: /database/index.db:/tmp/153048078
 wrote database to /tmp/153048078 <1>
 ...
-wrote mirroring manifests to manifests-{index-image}-1614211642
+wrote mirroring manifests to manifests-{index-image}-1614211642 <2>
 ----
 <1> Directory for the temporary `index.db` database generated by the command.
+<2> Note the name of the manifests directory that is generated.
+
+** *Option B:* If your mirror registry is on a completely disconnected, or airgapped, host, take the following actions.
+
+.. Run the following command on your workstation with unrestricted network access to mirror the content to local files:
++
+[source,terminal]
+----
+$ oc adm catalog mirror \
+    <index_image> \//<1>
+    file:///local/index \//<2>
+    [-a ${REG_CREDS}] \
+    [--insecure]
+----
+<1> Specify the index image for the catalog you want to mirror. For example, this might be a pruned index image that you created previously, or one of the source index images for the default catalogs, such as `{index-image-pullspec}`.
+<2> Mirrors content to local files in your current directory.
++
+.Example output
+[source,terminal]
+----
+...
+info: Mirroring completed in 5.93s (5.915MB/s)
+wrote mirroring manifests to manifests-my-index-1614985528 <1>
+
+To upload local images to a registry, run:
+
+	oc adm catalog mirror file://local/index/myrepo/my-index:v1 REGISTRY/REPOSITORY <2>
+----
+<1> Note the name of the manifests directory that is generated.
+<2> Note the file path given in the example command.
+
+.. Copy the `v2/` directory that is generated in your current directory to removable media.
+
+.. Physically remove the media and attach it to a host in the disconnected environment that has access to the mirror registry.
+
+.. Run the following command on the host in the disconnected environment to upload the images to the mirror registry:
++
+[source,terminal]
+----
+$ oc adm catalog mirror \
+    file://local/index/<repo>/<index_image>:<tag> \//<1>
+    <mirror_registry>:<port>/<namespace> \//<2>
+    [-a ${REG_CREDS}] \
+    [--insecure]
+----
+<1> Specify the file path that was given in the previous command output.
+<2> Specify the target registry and namespace to mirror the Operator content to, where `<namespace>` is any existing namespace on the registry. For example, you might create an `olm-mirror` namespace to push all mirrored content to.
+
+. After mirroring the content to the registry, inspect the manifests directory that is generated in your current directory. If you chose Option A in the previous step, the manifests directory name takes the following form:
```

Comment:
I already forgot which one was option A and which one was option B. 😂  I'd probably say something like "If your registry is attached to your network" and "If your registry is in a disconnected environment" instead.

I might also s/inspect/record or add a note at the end that you're going to need this directory name. IDK if I'm just not paying attention today, but when I got to the next step, I did not know that you meant this directory.

Reply:
I originally had it how you're suggesting, and switched it to referring just to the "Options" to try to shorten it up, but it's clear that is not helpful so will revert!